indico/modules/events/papers/templates/display/_paper_list.html

```diff
@@ -0,0 +1,61 @@
+                                </a>
+                            </div>
+                            <div class="row-content">
+                                <div class="row-left">
+                                    {% if reviewing -%}
+                                        <div class="submitter icon-user" data-searchable="{{ latest_revision.submitter.display_full_name|lower }}">
+                                            {{ latest_revision.submitter.display_full_name }}
+                                        </div>
+                                        <div>
+                                            {% if session.user in contribution.paper_layout_reviewers %}
+                                                <div class="label i-tag layout-reviewer review-role">
+                                                    {%- trans %}Layout{% endtrans -%}
+                                                    {% if latest_revision.has_user_reviewed(session.user, 1) %}
+                                                        <i class="icon-circle-small" title="{% trans %}Already reviewed{% endtrans %}"></i>
+                                                    {% endif %}
+                                                </div>
+                                            {% endif %}
+                                            {% if session.user in contribution.paper_content_reviewers %}
+                                                <div class="label i-tag content-reviewer review-role">
+                                                    {%- trans %}Content{% endtrans -%}
+                                                    {% if latest_revision.has_user_reviewed(session.user, 2) %}
+                                                        <i class="icon-circle-small" title="{% trans %}Already reviewed{% endtrans %}"></i>
+                                                    {% endif %}
+                                                </div>
+                                            {% endif %}
+                                        </div>
+                                    {% endif %}
+                                    <div class="label i-tag {{ latest_revision.state.name }}">
+                                        {{ latest_revision.state.title }}
+                                    </div>
+                                </div>
+                                <div class="row-right">
+{#                                    TODO: List attached files #}
+            {#                        {% for file in latest_revision.files %}#}
+            {#                            <a href="... link to the gfile...">{{ file.title }}</a><br>#}
+            {#                        {% endfor %}#}
+                                    <a>file_name.pdf</a><br>
+                                    <a>file_name.pdf</a><br>
+                                    <a>file_name.pdf</a><br>
```

Comment:
Well I implemented this displaying in Reviewing area which uses the same macro to render list, so once we merge that I will reabse and apply it here too ;)